orchagent/portsorch.cpp

```diff
@@ -2613,6 +2618,11 @@ bool PortsOrch::addLag(string lag_alias)
     lag.m_members = set<string>();
     m_portList[lag_alias] = lag;
 
+    vector<FieldValueTuple> fvVector;
+    FieldValueTuple s("state", "ok");
+    fvVector.push_back(s);
+    m_stateLagTable->set(lag_alias, fvVector);
```

Comment:
state ok is always linux kernel state ok, in this case, after you have created lag port in the linux kernel, you should say ok.

Reply:
should be in teamgr.cpp or if you want to use netlink message to decide whether lag has been created or not.

---
In reply to: [219434221](https://github.com/Azure/sonic-swss/pull/626#discussion_r219434221) [](ancestors = 219434221)